File: 2523/2523.c
int closestPrimeNumbersInRange(struct TreeNode* root) {
    if (!root) return 0;
    int l = closestPrimeNumbersInRange(root->left);
    int r = closestPrimeNumbersInRange(root->right);
    return 1 + (l > r ? l : r);
}

File: 2523/2523.py
class Solution:
    def closest_prime_numbers_in_range(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.closest_prime_numbers_in_range(root.left), self.closest_prime_numbers_in_range(root.right))